src/components/createPatientModal/createPatientModal.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+import Button from 'react-bootstrap/esm/Button'
+import Form from 'react-bootstrap/esm/Form'
+import Modal from 'react-bootstrap/esm/Modal'
+
```

Comment:
Does the name of this component make sense? Would something like `editPatientModal` be better suited?

Reply:
renamed to editPatientModal